src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -180,7 +181,9 @@ public SpawnResult exec(Spawn spawn, SpawnExecutionContext context)
     boolean acceptCachedResult = remoteExecutionService.getReadCachePolicy(spawn).allowAnyCache();
     boolean uploadLocalResults = remoteExecutionService.getWriteCachePolicy(spawn).allowAnyCache();
 
-    RemoteAction action = remoteExecutionService.buildRemoteAction(spawn, context);
+    RemoteAction action =
+        remoteExecutionService.buildRemoteAction(
+            spawn, context, MerkleTreeComputer.BlobPolicy.KEEP);
```

Comment:
This is slightly misleading since the policy may be overridden by `buildRemoteAction` according to `--experimental_remote_discard_merkle_trees`. Can we move the full computation to the call site?